### PR TITLE
Establish Learning Journey Working Group

### PR DESCRIPTION
**Pull request by zmitchell**
This PR establishes a Learning Journey Working Group with me as lead as requested by @fricklerhandwerk. I've also added myself as a member of the documentation team.

### CODE REVIEW

**Comment by zmitchell** [comment]
I have read the CLA Document and I hereby sign the CLA

File: working_groups/learning_journey.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+By extension the WG will be responsible for practical aspects such as:
+- Some portion of the structure of `nix.dev` so that documentation is discoverable and organized intuitively
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- TBD
```
Comment (fricklerhandwerk) [inline working_groups/learning_journey.md:18]: @olaf @asymmetric @wiseh @tonyfinn please add yourself as a suggestion if you're interested in participating.

File: maintainers/README.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ The team’s reason to exist is to make that principle discoverable and reproduc
   [@olafklingt]: https://github.com/olafklingt
   [@olaf]: https://discourse.nixos.org/u/olaf
 
+- Zach Mitchell (GitHub: [@zmitchell])
+
+  [@zmitchell]: https://github.com/zmitchell/
+
```
Comment (brianmcgee) [inline maintainers/README.md:50]:
```suggestion

- Brian McGee (Github: [@brianmcgee], Discourse: [@bmg])

  [@brianmcgee]: https://github.com/brianmcgee
  [@bmg]: https://discourse.nixos.org/u/bmg
```

File: working_groups/learning_journey.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+### Establish tooling and workflow
+We should have tooling that supports authors and enables them to contribute to the documentation with minimal boilerplate.
+
+Examples:
+- Macros or snippets to insert callouts ("Note:", "Warning:", etc)
+- etc
+
+## Long term projects
+### Migrate existing documentation
+There is a large body of existing documentation that should not be discounted. We should use the existing documentation to bootstrap the new
```
Comment (fricklerhandwerk) [inline working_groups/learning_journey.md:81]: Prior art: https://github.com/NixOS/nix-book/blob/main/resources.md

File: working_groups/learning_journey.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    - Flake schema
+
+### Set standards for documentation pages
+In order to provide a level of consistent quality for the documentation, we should decide on the bare minimum standards for documentation pages.
+
+Examples:
+- Pages should have a table of contents
+- Pages should refrain from first person voice
+- Decide how/whether different documentation sections should be handled differently (e.g. tutorial vs reference material)
+- etc
```
Comment (fricklerhandwerk) [inline working_groups/learning_journey.md:70]: Related:
- https://nix.dev/contributing/documentation
- https://github.com/NixOS/nix.dev/blob/master/CONTRIBUTING.md?plain=1#L47-L77
- https://github.com/NixOS/nix.dev/pull/311

File: working_groups/learning_journey.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    - The Nix Store and the database
+    - Channels, profiles, etc
+    - Overlays, overrides, `follows`
+- Reference
+    - Command reference
+    - How store paths are computed
+    - Build phases
+    - How derivations work
+    - Nix language reference
+    - Flake schema
```
Comment (fricklerhandwerk) [inline working_groups/learning_journey.md:61]: We did some [shuffling of topics and problems](https://pad.lassul.us/9f-dN2FyScmJ_OOMqAr_ew?both#) with @olaf yesterday in preparation for https://github.com/NixOS/nix.dev/issues/458 and should synchronise all that in the team meetings this week, and see which parts are most meaningful to get funding for.

Related: https://github.com/NixOS/nix.dev/pull/265

**Review by fricklerhandwerk — APPROVED** [review]
Awesome. Please link to this page from the `README` so it is discoverable.

File: maintainers/working_groups/learning_journey/README.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+By extension the WG will be responsible for practical aspects such as:
+- Some portion of the structure of `nix.dev` so that documentation is discoverable and organized intuitively
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- TBD
```
Comment (brianmcgee) [inline maintainers/working_groups/learning_journey/README.md:20]:
```suggestion
- @brianmcgee 
```

File: maintainers/README.md (hunk 2 above)
Comment (henrik-ch) [reply maintainers/README.md:50]: Hi @zmitchell, I meant to add myself to both. Shall I make a suggestion further down?

Comment (zmitchell) [reply maintainers/README.md:50]: There's a separate file for just the working group here: https://github.com/NixOS/nix.dev/pull/468/files#diff-aa46b0f178e55378224b0dd846934c5736fa2e4de6d18c20d2ab2d739f0f0567

You can make a suggestion to add yourself and I'll just accept it

File: maintainers/working_groups/learning_journey/README.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+- Some portion of the structure of `nix.dev` so that documentation is discoverable and organized intuitively
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- @brianmcgee 
+
```
Comment (henrik-ch) [inline maintainers/working_groups/learning_journey/README.md:21]:
```suggestion
- @henrik-ch
```

File: maintainers/README.md [hunk 8]
```diff
@@ -44,6 +44,13 @@ The team’s reason to exist is to make that principle discoverable and reproduc
...
```
Comment (evils) [inline maintainers/README.md:50]:
```suggestion

- Evils (GitHub: [@evils], Matrix: [@evils:nixos.dev])

 [@evils]: https://github.com/evils
 [@evils:nixos.dev]: https://matrix.to/#/@evils:nixos.dev
```

File: maintainers/working_groups/learning_journey/README.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+## Meetings
+Meetings will be held every week to ensure that progress can be made between meetings.
+
+## Immediate projects
+### Decide on documentation structure
+This project will decide on a high level structure for the documentation so that contributors know the audience they are speaking to and where their contributions fit into the Nix documentation landscape.
+
+The documentation will follow the [Diataxis](https://diataxis.fr) framework to the degree that it makes sense. This project will produce a proposal to be submitted to the Documentation Team.
+
+The first draft of this structure is as follows:
```
Comment (infinisil) [inline maintainers/working_groups/learning_journey/README.md:33]: I don't think it makes sense for this draft to be in the working group document, but I guess it's better than having it be nowhere. Just need to make sure to remove it from here when it's committed somewhere else

File: maintainers/working_groups/learning_journey/README.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- @brianmcgee 
+- @henrik-ch
+Membership is open to anyone that wants to contribute.
```
Comment (erooke) [inline maintainers/working_groups/learning_journey/README.md:22]:
```suggestion
- @erooke

Membership is open to anyone that wants to contribute.
```

**Pull request merged**